docs/reference/commandline/swarm_join_token.md

```diff
@@ -21,13 +21,14 @@ Options:
       --rotate   Rotate join token
 ```
 
-Join tokens are secrets that determine whether or not a node will join the swarm as a manager node
-or a worker node. You pass the token using the `--token flag` when you run
-[swarm join](swarm_join.md). You can access the current tokens or rotate the tokens using
-`swarm join-token`.
+Join tokens are secrets that determine whether or not a node will join the swarm
+as a manager node or a worker node. You pass the token using the `--token flag`
+when you run [swarm join](swarm_join.md). Nodes use the join token only when they join the swarm.
 
-Run with only a single `worker` or `manager` argument, it will print a command for joining a new
-node to the swarm, including the necessary token:
+You can access the current tokens or rotate the tokens using `swarm join-token`.
+
+Run with only a single `worker` or `manager` argument, it will print a command
```

Comment:
As a convenience, you can pass `worker` or `manager` as an argument to `join-token`, and docker will print the full command necessary to join a new node to the swarm.